bot/handlers/whois.py

```diff
@@ -8,6 +8,45 @@
 from users.models.user import User
 
 
+@is_club_member
+def whos_joined(update: Update, context: CallbackContext) -> None:
+
+    original_message = update.message  # look at the author of this message (works only in private chats)
+    from_user = original_message.from_user
+    from_username = original_message.chat.username
+
+    context.bot.delete_message(chat_id=update.message.chat_id, message_id=update.message.message_id)
+
+    if from_user.is_bot:
+        update.message.reply_text(
+            f"🤖В ч@т з@ш3л 60т: {from_user.id}",
+            quote=True
+        )
+        return None
+
+    telegram_id = from_user.id
+    user = User.objects.filter(telegram_id=telegram_id).first()
```

Comment:
я не уверен есть ли такое в питоне, но разве не должно быть так?
```suggestion
    user = User.objects.find(telegram_id=telegram_id)
```

Reply:
@kirillku, это Django ORM. Можно либо сделать `Column.objects.get(...)` и словить `Column.DoesNotExist` exception, либо `Column.objects.filter(...).first()` и получить `None`. Здесь выбран второй способ, он нормальный и часто используется.